src/core/redux/ReduxEpic.js

```diff
@@ -22,6 +22,8 @@ export default function reduxEpic() {
     PermissionsEpic,
 
     CreateOrganizationEpic,
-    OrganizationsEpic.fetchOrgsEpic
+    OrganizationsEpic.fetchOrgsEpic,
```

Comment:
Use `combineEpics` to merge these within `OrganizationsEpic` then register a single `OrganizationsEpic`. See `EdmEpic` for an example.